scripts/k8s/entrypoint.sh

```diff
@@ -56,9 +56,13 @@ if [ ! -z "$LABEL_SELECTOR" ]; then
 
 	_PODS=$(curl -s $CA -H "$AUTH" $PROTO://${HOST}:${PORT}/api/v1/namespaces/${POD_NAMESPACE}/pods?labelSelector=$LABEL_SELECTOR|jq .items[].status.podIP|sed -e "s/\"//g")
 	LIST=
+
 	for server in $_PODS; do
-	 LIST="${LIST}$server,"
+		if [ "$server" != "null" ] ; then
```

Comment:
have we observed a situation where K8s returned `null` here? was that a pod which just recently got created and didn't have an IP assigned yet?

Reply:
yes. i've seen this frequently. see #683

Reply:
and this shouldn't be an issue, because once that pod comes up it will obtain the IPs of the others, connect to them and become a part of the memberlist cluster, right?

Reply:
right